odoo/addons/base/models/ir_ui_view.py

```diff
@@ -2358,18 +2358,38 @@ def _get_default_form_view(self):
         :returns: a form view as an lxml document
         :rtype: etree._Element
         """
-        group = E.group(col="4")
+        sheet = E.sheet(string=self._description)
+        main_group = E.group()
+        left_group = E.group()
+        right_group = E.group()
         for fname, field in self._fields.items():
             if field.automatic:
                 continue
             elif field.type in ('one2many', 'many2many', 'text', 'html'):
-                group.append(E.newline())
-                group.append(E.field(name=fname, colspan="4"))
-                group.append(E.newline())
+                # append to sheet left and right group if needed
+                if len(left_group) > 0:
```

Comment:
Could be simplified, I think
```suggestion
                if left_group:
```